src/components/notifications/Cellsynt.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class="mb-3">
+        <label for="cellsynt-login" class="form-label">{{ $t("Username") }}</label>
+        <input id="cellsynt-login" v-model="$parent.notification.cellsyntLogin" type="text" class="form-control" required>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-key" class="form-label">{{ $t("Password") }}</label>
+        <HiddenInput id="cellsynt-key" v-model="$parent.notification.cellsyntPassword" :required="true" autocomplete="new-password"></HiddenInput>
+    </div>
+    <div class="mb-3">
+        <label for="cellsynt-Originatortype" class="form-label">{{ $t("Originator type") }}</label>
+        <select id="cellsynt-Originatortype" v-model="$parent.notification.cellsyntOriginatortype" :required="true" class="form-select">
+            <option value="alpha">{{ $t("Alphanumeric (recommended)") }}</option>
+            <option value="numeric">{{ $t("Telephone number") }}</option>
+        </select>
+        <div class="form-text">
+            <p><b>{{ $t("Alphanumeric (recommended)") }}:</b><br /> {{ $t("Alphanumeric string (max 11 alphanumeric characters). Recipients can not reply to the message.") }}</p>
+            <p><b>{{ $t("Telephone number") }}:</b><br /> {{ $t("Numeric value (max 15 digits) with telephone number on international format without leading 00 (example UK number 07920 110 000 should be set as 447920110000). Recipients can reply to the message.") }}</p>
+        </div>
+    </div>
+    <div class="mb-3">
+        <label v-if="$parent.notification.cellsyntOriginatortype === 'alpha'" for="cellsynt-originator" class="form-label">{{ $t("Originator") }} <small>{{ $t("(max 11 alphanumeric characters)") }}</small></label>
+        <label v-if="$parent.notification.cellsyntOriginatortype === 'numeric'" for="cellsynt-originator" class="form-label">{{ $t("Originator") }} <small>{{ $t("(max 15 digits)") }}</small></label>
+        <input v-if="$parent.notification.cellsyntOriginatortype === 'alpha'" id="cellsynt-originator" v-model="$parent.notification.cellsyntOriginator" type="text" class="form-control" pattern="[a-zA-Z0-9\s]+" maxlength="11" required>
+        <input v-if="$parent.notification.cellsyntOriginatortype === 'numeric'" id="cellsynt-originator" v-model="$parent.notification.cellsyntOriginator" type="number" class="form-control" pattern="[0-9]+" maxlength="15" required>
```

Comment:
Don't know which of these is more maintainable. What do you think?

```suggestion
        <input id="cellsynt-originator" v-model="$parent.notification.cellsyntOriginator" :type="$parent.notification.cellsyntOriginatortype === 'alpha' ? 'number' : 'text'" class="form-control" :pattern="$parent.notification.cellsyntOriginatortype === 'alpha' ? '[a-zA-Z0-9\s]+' : '[0-9]+'" :type="$parent.notification.cellsyntOriginatortype === 'alpha' ? 11 : 15" required>
```

```suggestion
        <input v-if="$parent.notification.cellsyntOriginatortype === 'alpha'" id="cellsynt-originator" v-model="$parent.notification.cellsyntOriginator" type="text" class="form-control" pattern="[a-zA-Z0-9\s]+" maxlength="11" required>
        <input v-else id="cellsynt-originator" v-model="$parent.notification.cellsyntOriginator" type="number" class="form-control" pattern="[0-9]+" maxlength="15" required>
```